### DISCUSSION TITLE
107 - как

### DISCUSSION THREAD

**Discussion by tarovik** [maxsite/cms#443] [General]
Была версия 105
обновился до 107
теперь global $page стала NULL
**КАК** узнать custom meta, присвоенные странице?
ранее было так 
$some_id = mso_page_meta('some_id', $page['page_meta'], '', '','', true);
теперь не работает

**Comment by maxsite** [comment]
Данные страницы так получаются:

`$pageData = mso_get_val('mso_pages', 0, true);`

https://max-3000.com/page/maxsite-cms-107

**Reply by tarovik** [reply]
создал для _данной_ страницы meta some_id
sql запрос SELECT * FROM `mso_meta` WHERE `meta_id_obj` = 2446
показывает, что есть some_id

вызов  mso_get_val('some_id', 0, true); из php блока _этой_ страницы показывает *bool(false).*

**Reply by maxsite** [reply]
Выполните, станет понятно:

```
$pageData = mso_get_val('mso_pages', 0, true);
pr($pageData);
```

**Reply by tarovik** [reply]
спасибо, 
нашёл в $pageData['page_meta']
теперь надо понять - как использовать (особенно, если эта мета не установлена)
пошёл читать исходники.
